code/_compile_options.dm

```diff
@@ -72,6 +72,17 @@
 #define TESTING
 #endif
 
+#ifdef TGS
+// TGS performs its own build of dm.exe, but includes a prepended TGS define.
+#define CBT
+#endif
+
 // A reasonable number of maximum overlays an object needs
 // If you think you need more, rethink it
 #define MAX_ATOM_OVERLAYS 100
+
+#if !defined(CBT) && !defined(SPACEMAN_DMM)
+#error "Building with Dream Maker is no longer supported."
+#error "In order to build, run BUILD.bat in the root directory."
+#error "Consider using Visual Studio Code instead, where you can Ctrl+Shift+B to build."
```

Comment:
Make this a warning, stop trying to stop people from making mistakes on their own computers.

Reply:
```suggestion
#warn "Building with Dream Maker is no longer supported."
#warn "In order to build, run BUILD.bat in the root directory."
#warn "Consider using Visual Studio Code instead, where you can Ctrl+Shift+B to build."
```

Reply:
Works for me, as long as it explicitly shows the footgun and they know what to do if something breaks before asking us.